Testing/SystemTests/tests/framework/DrillProcessTest.py

```diff
@@ -0,0 +1,160 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+# from Interface import *
+
+import systemtesting
+from unittest import mock
+import sys
+import time
+
+from qtpy.QtWidgets import QApplication, QFormLayout, QLineEdit, QComboBox
+from qtpy.QtTest import QTest
+from qtpy.QtCore import Qt, QPoint
+
+from mantid.kernel import config
+from mantid.simpleapi import mtd, GroupWorkspaces
+from Interface.ui.drill.view.DrillView import *
+from Interface.ui.drill.view.DrillSettingsDialog import *
+
+
+app = QApplication(sys.argv)
+
+
+class DrillProcessTest(systemtesting.MantidSystemTest):
```

Comment:
Can you name this DrillProcessSANSTest? We should gradually add those for each of the supported technique. We can add the classes in the same file.

Reply:
Done